### ISSUE TITLE
Справочники профессий и уровней.

### ISSUE THREAD

**Issue by mput** [Hexlet/hexlet-interview#75]
Поля profession и position должны стать enum type с перечисление возможных значений на манер interviewState.
Все шаблоны форм которые назначают эти поля должны использовать значения из типа.

**Comment by mput** [comment]
Или как вариант, вынести справачные значения в enum тип, который можно было бы импортировать из всех мест, где нужны эти данные.

**Comment by jougene** [comment]
думаю пока енумами обойдемся

**Comment by mput** [comment]
Да, я тоже думаю что этого достаточно. Но важно вынести эти данные в отдельное место, сейчас у нас в куча шаблонов справочники захардкодены.

**Comment by mput** [comment]
@jougene  Отредактировал изначальный issue. Добавь еще лабел goodfirst и frontend.
